WordPress/src/test/java/org/wordpress/android/ui/reader/repository/usecases/PostLikeUseCaseTest.kt

```diff
@@ -31,60 +48,273 @@ class PostLikeUseCaseTest {
 
     private lateinit var useCase: PostLikeUseCase
 
+    private val readerPost = createDummyReaderPost()
```

Comment:
We might want to consider re-creating the model in each test - using a shared state might bite us in the future as the order of the tests can affect the result. Wdyt?

Reply:
Same commit as above.